CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/issues) (the ones tagged
+  "help wanted" are usually a good place to start!)
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    and move to it with `cd <os-directory>`.
+
+4. Install dependencies:
```

Comment:
The Linux developments also needs to follow the dependencies listed for the module [dbus](https://www.npmjs.com/package/dbus#general)

Reply:
Is that something that could be automated by adding it to the `package.json`?

Reply:
No, because this is dependent on the distro they use. Each distro has different names for the modules. For example, the `libdbus-1-dev` dependency has [all this names](http://rosindex.github.io/d/libdbus-dev/) across distros.
I think just pointing to the `dbus` module page that has some example and they also mention "or equivalent for your system" which we can also mention.